src/components/domains/Domains.jsx

```diff
@@ -5,93 +5,89 @@ import { useLayoutEffect } from "react";
 
 const Domains = () => {
   return (
-    <div className="min-h-screen flex flex-col px-20 pt-32 bg-primaryBackground text-white">
-    
-    <div className="text-white min-h-screen overflow-x-hidden bg-primaryBackground">
-      <div className="sm:pb-14 ">
-        <div className="sm:pt-[200px]  flex md:flex-row flex-col  items-center justify-center space-x-10 pr-[80px]  sm:pr-[280px] ">
-          <h1 className="  uppercase relative md:rotate-90 rotate-0    tracking-widest sm:text-[100px] text-[80px] font-semibold">
-            Domains
-            <div className=" bg-red-600 rotate-90 w-2 h-[150px] absolute left-[450px] top-[70px]   " />
-          </h1>
-          <p className=" max-w-5xl  sm:text-3xl text-2xl  font-extralight  ">
-            Domains refer to different departments or areas of focus within the
-            club. These domains are responsible for organizing and executing
-            specific projects, activities, and initiatives that align with the
-            club's mission and goals.
-          </p>
+    <div className="min-h-screen flex flex-col px-2 sm:px-2 lg:px-20 pt-32 bg-primaryBackground text-white">
+      <div className="text-white min-h-screen overflow-x-hidden bg-primaryBackground">
+        <div className="sm:pb-14 ">
+          <div className="sm:pt-[200px]  flex md:flex-row flex-col  items-center justify-center space-x-10 pr-[80px]  sm:pr-[280px] ">
```

Comment:
```suggestion
          <div className="sm:pt-[200px]  flex md:flex-row flex-col  items-center justify-center space-x-10 sm:pr-[280px] ">
```

in mobile view it looks like this maybe the `pr-[80px]` is the thing or some thing else
<img width="540" alt="image" src="https://github.com/SCC-SRM/SCCWebsite/assets/59236972/555502ca-4cfa-4a3f-8f55-27e9a363d992">